### DISCUSSION TITLE
Suspension with no reason

### DISCUSSION THREAD

**Discussion by cp0317** [community/community#138856] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Today, one github account of mine was suspended, but I didn't get any comment or reason for that.
Please let me know where and what happened? on my account or on gitub?

PS: the account is very important. Please help me, folks

**Comment by JoshuaDevCo** [comment]
I have the same issue and waiting over 3 months already… they don't even respond on the tickets

**Comment by LiteBrite82** [comment]
Hi @cp0317 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by JoshuaDevCo** [reply]
i tried but they do not reply to me
